src/services/namingService.test.ts

```diff
@@ -86,4 +86,60 @@ describe("Naming Service", () => {
     const actual = AzureNamingService.getNormalizedRegionName(expected);
     expect(actual).toEqual(expected);
   });
+
+  it("deployment name is generated correctly", () => {
+    const config: ServerlessAzureConfig = {
+      functions: [],
+      plugins: [],
+      provider: {
+        prefix: "sls",
+        name: "azure",
+        region: "westus",
+        stage: "dev",
+      },
+      service: "test-api"
+    };
+
+    const timestamp = Date.now().toString();
+
+    const deploymentName = AzureNamingService.getDeploymentName(config, `t${timestamp}`);
+    expect(deploymentName.length).toBeLessThanOrEqual(64);
```

Comment:
replace all this with a call to `assertValidDeploymentName`

Reply:
Nice catch! - I had meant to do exactly that but didn't replace in all use cases.